blaze-client/src/test/scala/org/http4s/client/blaze/BlazeSimpleHttp1ClientSpec.scala

```diff
@@ -3,6 +3,7 @@ package org.http4s.client.blaze
 import org.http4s.client.ClientRouteTestBattery
 import org.http4s.util.threads.newDaemonPoolExecutionContext
 
+@deprecated("Well, we still need to test it", "0.18.0-M6")
```

Comment:
Is this to get around a deprecation warning invoking deprecated code?

I don't think we necessarily do need to test it, if this test coverage exists against the new interface.

Reply:
The pooled client is tested, as it was before. We are still providing the simple client, although it is deprecated. This is just to keep the tests for that running.

Should the deprecated simple client instead point to a default config pooled client?

Reply:
Let's leave it and not change behavior, in case people are relying on it.  We're already scolding the user.